src/main/java/org/cyclonedx/generators/xml/BomXmlGenerator.java

```diff
@@ -21,15 +21,14 @@
 import org.cyclonedx.CycloneDxSchema;
 import org.w3c.dom.Document;
 import javax.xml.parsers.ParserConfigurationException;
-import javax.xml.transform.TransformerException;
 
 public interface BomXmlGenerator {
 
     CycloneDxSchema.Version getSchemaVersion();
 
     Document generate() throws ParserConfigurationException;
 
-    String toXmlString() throws TransformerException;
+    String toXmlString() throws Exception;
```

Comment:
This is very generic, probably need something better, like a `GeneratorException`